cli/src/main.rs

```diff
@@ -60,7 +62,11 @@ async fn main() -> color_eyre::Result<()> {
             allow_existing,
         } => {
             let user_id = get_user_id_from_session()?;
-            append_url_to_db(&db_pool, &url, user_id, allow_existing).await?;
+            append_url(&db_pool, &url, user_id, allow_existing).await?;
+            // let outcome = append_url(&db_pool, &url, user_id, allow_existing).await?;
+            // if let AddUrlOutcome::Created(page) | AddUrlOutcome::Existing(page) = outcome {
+            //     process_page_snapshot(&db_pool, page.page_id, user_id, &url).await?;
+            // }
```

Comment:
The commented code is used for testing. Feel free to pull the PR and uncomment the code to see the output.

The output of the code looks something like this
<img width="1703" alt="Screenshot 2024-06-05 at 8 52 07 AM" src="https://github.com/coreyja/knowledge/assets/55955558/bedeb570-6e2f-4786-be3b-9970a8a1bff5">

Reply:
And actually looking at this code a bit more, I think we just want to add these lines un-commented!

Cause these are what hook up the downloading to the add_url cli command, which is exactly what we want for this ticket!

_if_ we wanted to comment something out we could comment out the println! that prints this all out, just cause it clutters the display. Buttt 🤷 this CLI is really just for us right now, and I think its nicer to see the HTML result as we keep going!

Reply:
Awesome! I will un comment it so we can use them!